test/unit/processing/payment-schedule/get-payment-schedule.test.js

```diff
@@ -208,4 +212,21 @@ describe('get various components and transform to payment schedule object', () =
     await getPaymentSchedule(paymentRequestId)
     expect(getScheduleDates).toHaveBeenCalled()
   })
+
+  test('should call getRemainingAmount when a paymentRequestId is given', async () => {
+    const paymentRequestId = 1
+    await getPaymentSchedule(paymentRequestId)
+    expect(getRemainingAmount).toHaveBeenCalled()
+  })
+
+  test('should call getRemainingAmount with scheduleDates when a paymentRequestId is given', async () => {
+    const newPaymentSchedule = {}
+    const previousPaymentSchedule = {}
+    const delta = 1
+    const schedule = getScheduleDates(previousPaymentSchedule, newPaymentSchedule, delta)
```

Comment:
We define what `getScheduleDates` returns on line 98, so it's just returning that.

We can get rid of all this and just have `expect(getRemainingAmount).toHaveBeenCalledWith((await getScheduleDates()))`